lib/trollop.rb

```diff
@@ -400,27 +608,8 @@ def educate(stream = $stdout)
 
       spec = @specs[opt]
       stream.printf "  %-#{leftcol_width}s    ", left[opt]
-      desc = spec.desc + begin
-        default_s = case spec.default
-        when $stdout   then "<stdout>"
-        when $stdin    then "<stdin>"
-        when $stderr   then "<stderr>"
-        when Array
-          spec.default.join(", ")
-        else
-          spec.default.to_s
-        end
+      desc = spec.format_description
```

Comment:
Maybe rename `format_desc` to something like `desc_with_default` - something more descriptive?

Or just pull out the `begin ... end` part as `default_desc` - and concatenate here.